store-implementation/federated-store/src/main/java/uk/gov/gchq/gaffer/federatedstore/FederatedStore.java

```diff
@@ -102,9 +111,21 @@
  * @see Graph
  */
 public class FederatedStore extends Store {
+    private static final Logger LOGGER = LoggerFactory.getLogger(Store.class);
+    private static final String FEDERATED_STORE_PROCESSED = "FederatedStore.processed.";
     private FederatedGraphStorage graphStorage = new FederatedGraphStorage();
     private Set<String> customPropertiesAuths;
     private Boolean isPublicAccessAllowed = Boolean.valueOf(IS_PUBLIC_ACCESS_ALLOWED_DEFAULT);
+    private static final List<Integer> ALL_IDS = new ArrayList<>();
```

Comment:
Consider not using constant case for mutable lists.

Reply:
This is global list held and modified by all classes, to understand and know which ID number have been used by other classes.

What would you consider using?

Reply:
Camel case for mutable class members.

Reply:
OOooooooh completely misunderstood  you.